kubernetes/linux/Dockerfile.multiarch

```diff
@@ -1,8 +1,8 @@
 # Default base images. If you update them don't forgot to update variables in our build pipelines. Default values can be found in internal wiki. External can use ubuntu 18.04 and golang 1.18.3
 ARG GOLANG_BASE_IMAGE=
-ARG CI_BASE_IMAGE=
+ARG MARINER_IMAGE=mcr.microsoft.com/cbl-mariner/base/core:2.0

```

Comment:
please check whether this also is not allowed. AFAIK you cannot have image references in Dockerfile without getting flagged

Reply:
> please check whether this also is not allowed. AFAIK you cannot have image references in Dockerfile without getting flagged

> isn't this only third-party images which are not from Microsoft?